tslint.json

```diff
@@ -9,7 +9,7 @@
     "no-var-requires": false,
     "object-literal-sort-keys": false,
     "curly": false,
-    "no-unused-expression": false,
+    "no-unused-expression": [true, "allow-tagged-template"],
```

Comment:
I had to add this extra configuration to allow using tagged template in order to get around this:

```
ERROR: src/modules/common/styles/global-styles.ts[4, 1]: unused expression, expected an assignment or function call
```

https://github.com/erxes/erxes/blob/0cbbf446c97e678399a84dfd1b207c0d70f17eba/src/modules/common/styles/global-styles.ts#L4-L6

More on this: https://palantir.github.io/tslint/rules/no-unused-expression/

Reply:
Alternatively, I could remove this configuration entirely and use something like `tslint:disable-next-line` in the file itslef. What do you think?